article.html

```diff
@@ -0,0 +1,607 @@
+                      </div>
+                  </div>
+                  <div class="image img-wrap">
+                    <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_1715762034/container_1796181454/tabs/item_1669125273755/image_copy.coreimg{.width}.png/1669126312045/quote-img.png" data-asset="/content/dam/LeaderImpact/site-./assets/Rectangle_726.png" data-title="lady smiling" id="image-6bcf13ac84" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                      <img src="./assets/Rectangle_709.png" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt." title="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt." >
+                      <span class="cmp-image__title" itemprop="caption">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</span>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- ARTICLE SLIDER -->
+        <div class="container responsivegrid article-slider aem-GridColumn aem-GridColumn--default--12 layout-grid">
+          <div id="container-b0c5fbd9e2" class="cmp-container">
+            <div class="aem-Grid aem-Grid--12 aem-Grid--default--12 ">
+              <!-- ARTICLE MAIN TITLE -->
+              <div class="responsivegrid aem-GridColumn title-btn-wrap aem-GridColumn--default--12 layout-grid">
+                <div id="container-56884b3bb11" class="cmp-container">
+                  <!-- ARTICLE TITLE -->
+                  <div class="title aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="title-bcb754f8400" class="cmp-title">
+                      <h5 class="cmp-title__text">Leading Through a Crisis Video Series</h5>
+                    </div>
+                  </div>
+                </div>
+              </div>
+              <!-- ARTICLE SLIDER WRAP -->
+              <div class=" responsivegrid article-slider-wrap aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                <div id="container-9ce1cf14410" class="cmp-container">
+                  <!-- ARTICLE SLIDER ITEM -->
+                  <div class="container responsivegrid post-item aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                    <div id="container-172911e9c5" class="cmp-container">
+                      <div class="teaser aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div class="cmp-teaser">
+                          <div class="cmp-teaser__image post-article__image">
```

Comment:
@Prystal254 `post-article__image` can't go here. All custom classes will be declared on the root element of the component, line 370 in this case. This goes for line 394, `post-book__image` on line 416, and `post-video__image` on line 438.

Reply:
@Prystal254 According to the [documentation](https://www.aemcomponents.dev/content/core-components-examples/library/core-content/teaser.html#tabs-dcd5467079-item-84723062b6-tab), it looks like the image is defined after `.cmp-teaser__content`.

Reply:
is this wrong? by changing the position of the element... like I did on teaser content?

Reply:
called the icons images on cmp-image__link as they are being called in psuedo element.

Reply:
@Prystal254 My apologies. I thought I got rid of all my notes about the order of teaser content and images. We're not using the latest version of the teaser component, so the `.cmp-teaser__image` does indeed come before the `.cmp-teaser__content`. You are correct. If you see this note about switching their order anywhere else, ignore it.

All custom classes, like `.post-article__image` here, must be declared on the root element of the component. In this case, the root is on line 370. This same issue appears on lines 394, 416, and 438.